pkg/metrics/metrics.jsx

```diff
@@ -382,19 +382,25 @@ class CurrentMetrics extends React.Component {
             const merged = [];
             names.forEach((k, i) => {
                 const v = values[i];
+                let user;
                 // filter out invalid values, the empty (root) cgroup, non-services
                 if (k.endsWith('.service') && typeof v === 'number' && v != 0) {
+                    if (k.match(/^user.*user@\d+\.service.+/)) {
+                        user = true;
+                    } else {
+                        user = false;
+                    }
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v]);
+                    merged.push([label, v, user]);
```

Comment:
You also need to adjust the comment at the top of the function to document the new third item in the returned list.